src/request/components/request-detail-panel-logging.jsx

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+var messageProcessor = require('../util/request-message-processor');
+
+var moment = require('moment');
+var _ = require('lodash');
+var React = require('react');
+
+/**
+ * Return the log-write messages for the current request.
+ */
+var getPayloads = (function() {
+    var getItem = messageProcessor.getTypePayloadList;
+
+    var options = {
+        'log-write': getItem,
+    };
+
+    return function(request) {
+        return messageProcessor.getTypeStucture(request, options);
+    }
+})();
+
+/**
+ * Return the CSS class name to use for the given message
+ */
+function getRowClass(message) {
+    var rowClass = 'request-logs-data-default';
+    switch (message.level) {
+        case 'Verbose':
+        case 'Info':
+            rowClass = 'request-logs-data-default';
+            break;
+        case 'Critical':
+        case 'Error':
+            rowClass = 'request-logs-data-error';
+            break;
+        case 'Warning':
+            rowClass = 'request-logs-data-warning';
+            break;
+        default:
+            rowClass = 'request-logs-data-default';
+            break;
+    }
+    return rowClass;
+}
+
+/**
+ * React class to display console messages
+ */
+var LogMessages = React.createClass({
+    render: function() {
+        return (
+            <div>
+                <div>
+                    <div>
+                        <table>
+                            <col width="5%"/>
+                            <col width="10%"/>
+                            <col width="12%"/>
+                            <col width="13%"/>
+                            <col width="60%"/>
```

Comment:
You should be able to add these `width` attributes to the header `td`s and get the same effect. It will just make things a little more terse.